Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/) for Kodi add-ons: look and feel/visualization.milkdrop2.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-look-and-feel/-/visualization-milkdrop2/horizontal-auto.svg)

gade01 commented on visualization.milkdrop2/resources/language/resource.language.af_za/strings.po:
```diff
@@ -23,11 +23,3 @@ msgstr "MilkDrop laat jou sweef deur die klankgolwe wat jy hoor"
-#[Referencing] If a suitable string already exists, reuse it, making a note of where it's used!
-#[Description / location] For example, "#. Description of some setting" used on "#: xbmc/addons/guidialogaddoninfo.cpp"
-#When writing a description or setting, refer to a setting name in quotes. See existing entries for guidance.
-#For example, "Press \"OK\" for \"All seasons\"" instead of "Press OK for All seasons" after first word.
```
@AlwinEsch Is it correct of Weblate to remove these lines?

AlwinEsch replied:
Not really, specially as it bring in on new files, no idea whats there happen.

If them away is also not problematic, was come in by me with c&p from another addon.

gade01 replied:
Yeah, I saw that Weblate removes the lines in existing files and adds the exact same lines in new files.
What should we do?

AlwinEsch replied:
Think I create a request where removes them on all files and thing on next update here it goes away complete

gade01 replied:
Ah, great.
I forced a refresh on a Weblate addon and that seems to remove the lines from all languages now.

gade01 replied:
It force pushed to this PR.

AlwinEsch replied:
Thanks a lot! Then we use this and I change my other request to Nexus and takes the update commits from here.

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Indonesian (id_id))

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 50.0% (1 of 2 strings)

Translated using Weblate (English (Australia) (en_au))

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (2 of 2 strings)

Added translation using Weblate (Persian (Iran) (fa_ir))

Added translation using Weblate (Burmese (my_mm))

Added translation using Weblate (Uzbek (uz_uz))

Added translation using Weblate (Serbian (latin))

Added translation using Weblate (Serbian (sr_rs))

Added translation using Weblate (Maltese (mt_mt))

Added translation using Weblate (Malay (ms_my))

Added translation using Weblate (Maori)

Added translation using Weblate (Indonesian (id_id))

Added translation using Weblate (Hindi (India) (hi_in))

Added translation using Weblate (Hebrew (Israel) (he_il))

Added translation using Weblate (Faroese (fo_fo))

Added translation using Weblate (Persian (Afghanistan) (fa_af))

Added translation using Weblate (Basque (Spain) (eu_es))

Added translation using Weblate (Esperanto)

Added translation using Weblate (English (New Zealand) (en_nz))

Added translation using Weblate (English (Australia) (en_au))

Added translation using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Added translation using Weblate (Belarusian (be_by))

Added translation using Weblate (Azerbaijani (az_az))

Added translation using Weblate (Amharic (Ethiopia) (am_et))

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/en_au/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/en_nz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-milkdrop2/th_th/
Translation: Kodi add-ons: look and feel/visualization.milkdrop2
